src/tiles/grid/item-tile.ts

```diff
@@ -78,12 +80,24 @@ export class ItemTile extends LitElement {
    * Templates
    */
   private get creatorTemplate(): TemplateResult | typeof nothing {
-    if (!this.model?.creator) return nothing;
+    let displayedCreator = this.model?.creator;
+
+    // If we're filtering by creator initial and have multiple creators, we want
+    // to surface the first creator who matches the filter.
+    if (this.selectedCreatorFilter && this.model?.creators.length) {
+      const firstLetter = this.selectedCreatorFilter;
```

Comment:
I'd drop the `firstLetter` declaration ; `this.selectedCreatorFilter` is pretty clear.

Reply:
I agree in principle, but TypeScript can't guarantee `this.selectedCreatorFilter` isn't undefined within the `find` callback (_technically_ correct, I suppose). So it was either this or an explicit type assertion of some sort.

Reply:
ahhhh gotcha didn't think of that! Makes sense